src/components/dialog/theme.css

```diff
@@ -33,6 +33,7 @@
     transform var(--animation-duration) var(--animation-curve-default);
   transition-delay: var(--animation-delay);
   width: 100vw;
+  z-index: 2;
```

Comment:
Q: Is working with low values like these smart? Maybe keep some more breathing-space? Just thinking out loud (so keep the bigger values?).

Reply:
It's a relative value. Once a wrapper has `position: relative, absolute or fixed`, the inner z-index values are being added up to the parent value. In this case `.dialog-base` will have a z-index of `400 + 2`.

You will also notice that the Overlay component always have a `z-index: 1`.

Dialog wrapper: 400
>   Overlay: 1
>   Dialog: 2